### ISSUE TITLE
More than 100 items?

### ISSUE THREAD

**Issue by ShadowRunSucks** [pbrink231/plex_top_playlists#23]
It it possible to sync a imdb list with more than 100 items or is that the limit?

**Comment by pbrink231** [comment]
I dont think there is a limit.  Whatever page you load its however many items are there.

This page: https://www.imdb.com/chart/top has 250.

Do you have an example where the page is paginated?

**Comment by pbrink231** [comment]
going to close this.  if you find an example please let me know.

**Comment by hbooo** [comment]
Got the problem with this playlist : 

```
1001 Movies You Must See Before You Die: STARTING PLAYLIST - TYPE: custom - URL: https://www.imdb.com/list/ls052535080/?sort=release_date,desc&st_dt=&mode=detail&page=1
1001 Movies You Must See Before You Die: finding matching movies for playlist with count 100
```

There is only 100 movies by page

**Comment by pbrink231** [comment]
I see the page in the url.  Could try removing that and see if it shows all.  Small chance.

Will look into this to deal with paginated view when I can.  Traveling for a bit

**Comment by pbrink231** [comment]
Can you paste your settings line for this one here.

**Comment by hbooo** [comment]
I have this : 

`{ "title":"1001 Movies You Must See Before You Die", "type":"custom", "url":"https://www.imdb.com/list/ls052535080/?sort=release_date,desc&st_dt=&mode=detail&page=1", "kind":"collection"},`

but got the same result with 
`{ "title":"1001 Movies You Must See Before You Die", "type":"custom", "url":"https://www.imdb.com/list/ls052535080/?sort=release_date,desc&st_dt=&mode=detail", "kind":"collection"},`

**Comment by pbrink231** [comment]
Fixed this on custom IMDB lists.

you can check using this test code
`python .\plex_playlist_update.py imdb_ids "https://www.imdb.com/list/ls052535080/?sort=release_date,desc&st_dt=&mode=detail&page=1" custom`

The page does not have to be there in order for it to work.

fixed in version 2.03
